smartcar/requester.py

```diff
@@ -15,9 +16,14 @@ def call(method, url, **kwargs):
         dict: response from the request to the Smartcar API
 
     """
-    if not 'headers' in kwargs:
+    if 'headers' not in kwargs:
         kwargs['headers'] = {}
-    kwargs['headers']['User-Agent'] = 'smartcar-python-sdk:{}'.format(__version__)
+    kwargs['headers']['User-Agent'] = 'Smartcar/{} ({}; {}) Python v{}'.format(
```

Comment:
Might be a nit, but generally Python versions are without v. Python 2.7/Python 3.6 is the standard format. Might be weird for Python devs to add the v. Thoughts

Reply:
ah I see, hmm I just added that because node's `process.version` prints out `v8.11.x` and I wanted the string to be consistent across the 3 SDKs, I could however strip it out of the Node.js one

What do you think?

Reply:
I think we should stay consistent with the language because it would be what the devs are used to. So, I'd suggest ```v``` for Node and not for Python. I don't think anyone would store the ```user-agent``` to use across SDKs? I could be wrong about that, but if not, remaining consistent with language would take higher precedence for me. Kinda like documentation consistency we were talking about yesterday.

Reply:
I would say that in the case consistency for the language doesn't matter as much because this information is meant for us and not for the developers as this user-agent is just attached to the request that gets sent to our api and is not surfaced anywhere else. We should either have the `v` in all the SDKs or none because otherwise we'd have to have two sets of regexes for parsing user-agents

Reply:
I see, I'm not sure tbh. It makes it easier for us for sure, but it's not language consistent. But, I'm okay with approving this. I'd like to hear someone else's opinion too (@sankethkatta @esonmez  @GregChan). I'll approve it for now though.

Reply:
I'll wait to merge until this is resolved

Reply:
Since developers are not going to use this (even if they see it in the code), using a header that's consistent for us internally across SDKs makes sense to me.

Reply:
I don’t think language consistency matters as much because we’re the primary consumers of this. I have no preference for keeping or removing the v. My only requirement is that it should be easy to parse.

Reply:
Sounds good. Push! https://media.giphy.com/media/mEtSQlxqBtWWA/giphy.gif